pet-finder/src/Components/Forms/LoginForm/index.tsx

```diff
@@ -1,11 +1,19 @@
-import { FC } from 'react'
+import { FC, useCallback } from 'react'
 import { SubmitHandler, useForm } from 'react-hook-form'
 import { yupResolver } from '@hookform/resolvers/yup'
 
 import { StyledInput } from '../../StyledInput'
 import { StyledButton } from '../../StyledButton'
 
+import { logIn } from '../../../requests/req'
 import { LoginFormType, loginSchema } from '../../../utils/loginFormSchema'
+import { ERROR_MESSAGES } from '../../../constants/errorMessages'
+import { IResponseBodyLogIn } from '../../../requests/interfaces'
+import {
+  exceptionResponse,
+  ExceptionMessage,
+} from '../../../requests/constants'
+import { loginFormFields } from '../../../constants/formFields'
 
 import styles from './LoginForm.module.scss'
 
```

Comment:
empty interface

Reply:
4c2728b066835430aa3ca0889233c9a452184030
This interface has been removed